src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorPluginState.java

```diff
@@ -17,7 +17,7 @@
     private final WaifuMotivatorState state = new WaifuMotivatorState();
 
     public static WaifuMotivatorPluginState getInstance() {
-        return ServiceManager.getService( WaifuMotivatorPluginState.class );
+        return ApplicationManager.getApplication().getService( WaifuMotivatorPluginState.class );
```

Comment:
I've been having issues with this returning null.

I think it may have to do with the alarm debouncer.

When I pulled up the source code there was a nice little java doc

```
/**
 * For old-style components, the contract specifies a lifecycle: the component gets created and notified during the project opening process.
 * For services, there's no such contract, so we don't even load the class implementing the service until someone requests it.
 *
 * In a new code please use {@link ComponentManager#getService(Class)} or {@link ComponentManager#getServiceIfCreated(Class)}.
 */
```
So I switched it.